fc/protocol.py

```diff
@@ -135,14 +148,28 @@ def __init__(self, protoFile, indentLevel=0):
                 self.postProcessing.extend(imported_proto.postProcessing)
                 self.outputs.extend(imported_proto.outputs)
                 self.plots.extend(imported_proto.plots)
+                self.model_interface.extend(imported_proto.model_interface)
+                for prefix, uri in imported_proto.ns_map.items():
```

Comment:
We probably ought to have some error thrown if a prefix ends up getting bound to a different URI here!

Reply:
Good point! How do we raise errors? Do we have some kind of special Exception type that gets converted to user-friendly messages?

Reply:
Good point. Have added a test for this